tabbycat/importer/forms.py

```diff
@@ -150,6 +152,11 @@ class Meta:
         model = Venue
         fields = ('name', 'priority')
 
+    def save(self, *args, **kwargs):
+        venue = super().save(*args, **kwargs)
+        VenueIdentifier.objects.create(venue=venue)
+        return venue
```

Comment:
Should use the same signature as the other `save()` methods:
```suggestion
    def save(self, commit=True):
        venue = super().save(commit=False)
        if commit:
            VenueIdentifier.objects.create(venue=venue)
        return venue
```